experiments/Atmos_LES/bomex.jl

```diff
@@ -0,0 +1,502 @@
+  w′θ′   = FT(8e-3)     # Potential temperature flux
+  w′qt′  = FT(5.2e-5)   # Total moisture flux
+
+  bc     = BOMEX_BC{FT}(u_star, w′θ′, w′qt′) # Boundary conditions
+  
+  ∂qt∂t_peak = FT(-1.2e-8)  # Moisture tendency (energy source)
+  zl_qt = FT(300)           # Low altitude limit for piecewise function (moisture source)
+  zh_qt = FT(500)           # High altitude limit for piecewise function (moisture source)
+  ∂θ∂t_peak  = FT(-2/day)   # Potential temperature tendency (energy source)
+
+  z_sponge = FT(2400)       # Start of sponge layer
+  α_max    = FT(1.0)        # Strength of sponge layer (timescale)
+  γ        = 2              # Strength of sponge layer (exponent)
+  u_geostrophic = FT(-10)        # Eastward relaxation speed
+  u_slope       = FT(1.8e-3)     # Slope of altitude-dependent relaxation speed
+  v_geostrophic = FT(0)          # Northward relaxation speed
+
+  zl_sub = FT(1500)         # Low altitude for piecewise function (subsidence source)
+  zh_sub = FT(2100)         # High altitude for piecewise function (subsidence source)
+  w_sub  = FT(-0.65e-2)     # Subsidence velocity peak value
+
+  f_coriolis = FT(0.376e-4) # Coriolis parameter
+  
+  # Assemble source components
+  source = (
+            Gravity(),
+            BomexTendencies{FT}(∂qt∂t_peak, zl_qt, zh_qt, ∂θ∂t_peak, zl_sub, zh_sub, w_sub, zmax),
+            BomexSponge{FT}(zmax, z_sponge, α_max, γ, u_geostrophic, u_slope, v_geostrophic),
+            BomexGeostrophic{FT}(f_coriolis, u_geostrophic, u_slope, v_geostrophic)
+           )
+
+  # Assemble timestepper components
+  ode_solver_type = CLIMA.DefaultSolverType()
+
+  # Assemble model components
+  model = AtmosModel{FT}(AtmosLESConfiguration;
+                         turbulence        = SmagorinskyLilly{FT}(C_smag),
+                         moisture          = EquilMoist{FT}(;maxiter=15,
+                                                             tolerance=0.1),
+                         source            = source,
+                         boundarycondition = bc,
+                         init_state        = ics)
+  
+  # Assemble configuration
+  config = CLIMA.Atmos_LES_Configuration("BOMEX", N, resolution,
+                                         xmax, ymax, zmax,
+                                         init_bomex!,
+                                         solver_type=ode_solver_type,
+                                         model=model)
+    return config
+end
+
+function main()
+  CLIMA.init()
+
+  FT = Float64
+
+  # DG polynomial order
+  N = 4
+  # Domain resolution and size
+  Δh = FT(100)
+  Δv = FT(40)
+
+  resolution = (Δh, Δh, Δv)
+  
+  # Prescribe domain parameters
+  xmax = 6400
+  ymax = 6400
+  zmax = 3000
+
+  t0 = FT(0)
+  timeend = FT(3600*6)
+  CFLmax  = FT(0.5)
+
+  driver_config = config_bomex(FT, N, resolution, xmax, ymax, zmax)
+  solver_config = CLIMA.setup_solver(t0, timeend, driver_config, forcecpu=true, Courant_number=CFLmax)
```

Comment:
Do we want `forcecpu=true` as the default?

Reply:
`forcecpu` forces initialisation on CPU. In this instance I'm using random numbers in the initial condition so I need the CPU init argument to be true.

Reply:
This is fixed in #811